server/adaptors/integrations/__data__/repository/amazon_networkfirewall/assets/example_queries-1.0.0.ndjson

```diff
@@ -0,0 +1,4 @@
+{"attributes":{"createdTimeMs":1743385084317,"savedQuery":{"data_sources":"[{\"name\":\"test_data_source\",\"type\":\"s3glue\",\"label\":\"test_data_source\",\"value\":\"test_data_source\"}]","description":"","name":"Total bytes going to a domain","query":"SELECT event.http.hostname AS domain, SUM(event.netflow.bytes) AS total_bytes FROM {table_name} WHERE event.http.hostname IS NOT NULL GROUP BY event.http.hostname ORDER BY total_bytes DESC;","query_lang":"SQL","selected_date_range":{"end":"now","start":"now-15m","text":""},"selected_fields":{"text":"","tokens":[]},"selected_timestamp":{"name":"","type":"timestamp"}},"title":"Total bytes going to a domain","version":1},"id":"ca244a60-0dd0-11f0-99ac-779befa6084f","references":[],"type":"observability-search","updated_at":"2025-03-31T01:38:04.418Z","version":"WzgxNiwxXQ=="}
```

Comment:
nit: I think the original idea for these sample queries, is letting user perform these search with the skipping index we created. So maybe we can construct the query that will actually use the fields in skipping index.

Reply:
Yes, and I don't see this as a blocker for this PR. I was trying to call out that was the intention for adding the skipping index query.